src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/ReferralReportData.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import java.time.Instant
+import java.util.UUID
+
+interface ReferralReportData {
```

Comment:
this isn't an entity (no @Entity annotation). it's also identical to the DTO. can we just have one class to represent this thing?

Reply:
As far as we could see, In order for the native query to work and map, you have to use an interface.

Reply:
i see. i also now notice there are some differences in the types. why do we have to use Instant here ? our time stamp sql values map directly to OffsetDateTime elsewhere, why not here?

regarding this not being an entity, we should just define this class where it's used, since having it in the entity directory suggests this class represents a model in the database.

Reply:
Again, this was to get the mapping's to work properly. The sql would only seem to return as an SQL Timestamp and would not map to offsetDateTime directly

Reply:
this is the reason JPA exists 😝